default_config.yaml

```diff
@@ -1,4 +1,20 @@
+#
+# Copyright 2020 The dNation Jsonnet Translator Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 target_namespace: "default"
+source_namespace: "ALL"
```

Comment:
I'm perfectly fine with "ALL" if you prefer this naming, but maybe you could consider following suggestions: 
"ALL" -> "*"
 or 
"ALL" -> "any"